File: src/main/java/com/somefrills/commands/FreecamCommand.java
package com.somefrills.commands;

import com.mojang.brigadier.builder.LiteralArgumentBuilder;
import com.somefrills.features.core.Features;
import com.somefrills.features.misc.Freecam;
import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;

public class FreecamCommand {
    public static LiteralArgumentBuilder<FabricClientCommandSource> getBuilder() {
        return LiteralArgumentBuilder.<FabricClientCommandSource>literal("freecam")
                .executes(ctx -> {
                    Features.get(Freecam.class).toggle();
                    return 1;
                });
    }
}

File: src/main/kotlin/com/somefrills/commands/FreecamCommand.kt
package com.somefrills.commands

import com.mojang.brigadier.builder.LiteralArgumentBuilder
import com.mojang.brigadier.context.CommandContext
import com.somefrills.features.misc.Freecam
import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource

object FreecamCommand {
    fun getBuilder(): LiteralArgumentBuilder<FabricClientCommandSource> {
        return LiteralArgumentBuilder.literal<FabricClientCommandSource>("freecam")
            .executes { ctx: CommandContext<FabricClientCommandSource> ->
                Freecam.toggle()
                1
            }
    }
}